src/main/java/org/opensearch/security/configuration/DlsFlsValveImpl.java

```diff
@@ -191,11 +195,20 @@ public boolean invoke(PrivilegesEvaluationContext context, final ActionListener<
             }
         }
 
+        EvaluatedDlsFlsConfig dlsFlsConfigHeaders = filteredDlsFlsConfig;
+        if (clusterInfoHolder.getMinimumNodeVersion().onOrBefore(Version.V_2_19_2)) {
```

Comment:
how would ```V_2_19_2``` resolve in managed service?

Reply:
See https://github.com/opensearch-project/security/pull/5482#discussion_r2213100002